package.json

```diff
@@ -28,17 +27,29 @@
 		"webpack-cli": "^4.9.2"
 	},
 	"peerDependencies": {
+		"@types/bootstrap": "^5.2.4",
+		"@types/jquery": "^3.5.14",
+		"@types/sanitize-html": "^2.6.2",
```

Comment:
The type definitions for sanitize-html should be peer-level dependencies if sanitize-html itself is one.

Reply:
You are absolutely right, I get it.
With the type definitions as a `devDependency`, I can cleanly `npm install` and `npm run build`, great.